bin/alluxio

```diff
@@ -370,15 +287,30 @@ case ${COMMAND} in
   exit 0
 ;;
 "version")
-  CLASS=alluxio.Version
+  CLASS="alluxio.cli.Version"
 ;;
 "validateConf")
-  CLASS=alluxio.ValidateConf
+  CLASS="alluxio.cli.ValidateConf"
 ;;
 *)
+  echo "Unsupported command ${COMMAND}"
   printUsage
   exit 1
 ;;
 esac
 
-${JAVA} -cp ${CLASSPATH} ${ALLUXIO_USER_JAVA_OPTS} ${CLASS} ${PARAMETER} $@
+while getopts ":D:" opt; do
```

Comment:
factor this into a function and only call it for commands that will end up calling `java`